deployment/proxies.rst

```diff
@@ -31,8 +31,9 @@ and what headers your reverse proxy uses to send information::
 
     // tell Symfony about your reverse proxy
     Request::setTrustedProxies(
-        // the IP address (or range) of your proxy
-        ['192.0.0.1', '10.0.0.0/8'],
+        // you can use IP address, IP address ranges or the 'REMOTE_ADDR' string,
+        // which is replaced at run time by the $_SERVER['REMOTE_ADDR'] env var
```

Comment:
it's not an env var but a server variable
should we rephrase this to tell what this means in practice?

Reply:
I would be inclined to not include it in this part of the example and only have it in the next part. Just to avoid people copying stuff that is potentially insecure.

Reply:
💯 agree with the risk, we should explain it clearly to readers